draft-schwartz-add-ddr-forwarders.md

```diff
@@ -49,175 +49,192 @@ informative:
+By this process, Do53 is replaced with encrypted DNS for most queries. The client may wish to continue to send locally-relevant queries (e.g. .local) towards the legacy DNS forwarder.
 
 ## Scope
 
-This informational document describes the interaction between DDR and legacy DNS forwarders.  It discusses possible client policies, problems that might arise, and relevant mitigations.
+This document describes the interaction between DDR and legacy DNS forwarders.
 
 DNS forwarders and resolvers that are implemented with awareness of DDR are out of scope, as they are not affected by this discussion (although see Security Considerations, {{security-considerations}}).
 
 IPv6-only networks whose default DNS server has a Global Unicast Address are out of scope, even if this server is actually a simple forwarder.  If the DNS server does not use a private IP address, it is not a "legacy DNS forwarder" under this draft's definition.
 
-# Relaxed Validation client policy {#client-policy}
+# Conventions and Definitions
 
-We define a "relaxed validation" client policy as a client behavior that removes the certificate validation requirement when the Unencrypted Resolver is identified by a private IP address, regardless of the Designated Resolver's IP address.  Instead, under this condition, the client connects using the Opportunistic Privacy Profile of encrypted DNS ({{?RFC7858, Section 4.1}}).
+Private IP Address - Any IP address reserved for loopback {{?RFC1122}}, link-local {{?RFC3927}}, private {{?RFC1918}}, local {{?RFC4193}}, or Carrier-Grade NAT {{?RFC6598}} use.
 
-The Opportunistic Privacy Profile is a broad category, including clients that "might or might not validate" the TLS certificate chain even though there is no authentication identity for the server.  This kind of validation can be valuable when combined with a reputation system or a user approval step (see {{reputation}} and {{user-controls}}).
+Legacy DNS Forwarder - An apparent DNS resolver, known to the client only by a private IP address, that forwards the client's queries to an upstream resolver, and has not been updated with any knowledge of DDR.
 
-This client policy is otherwise identical to the one described in {{Section 4 of DDR}}.
+Cross-Forwarder Upgrade - Establishment and use of a direct, encrypted connection between the client and the upstream resolver.
 
-# Naturally compatible behaviors
+# Reputation Verified Selection (RVS) {#client-policy}
 
-The following system behaviors are naturally compatible with relaxed validation.
+On receipt of an answer to the _dns.resolver.arpa query, the client scans the available access methods and for each one extracts the offered Authentication Domain Name (ADN). Each ADN is then checked to see if it has sufficient reputation, according to a reputation scheme (e.g. a locally stored list of respectable resolvers). The particular choice of scheme is not prescribed (see {{reputation-systems}}) and is a decision for the client.
```

Comment:
I'll have to think about this tomorrow.

Reply:
Ben, I don't see why this is necessary!

Firstly there is an ADN in DDR. Section 7 of svcb-dns says "A resolver known as resolver.example that supports: DoT on resolver.example ports 853 (implicit in record 1) and 8530 (explicit in record 2), with "resolver.example" as the Authentication Domain Name"  This ADN comes from the TargetName of the SVCB record. (We probably ought to clarify that ADN appears as the TargetName.)

Second, from the client's perspective it will be trying to reach, and authenticate, a single ADN. Just because the server's TLS certificate supports "search.example" and "tunnelling.example" in addition to "resolver.example", that doesn't mean the client needs to pay any attention to that. The client should only care about resolver.example, and whether that name is reputable or not. If the name is not reputable, it should not connect, end of story.

If you think I'm wrong, please explain why?